docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/BinaryClassification/Calibrators/PairAdjacentViolators.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Linq;
+
+namespace Microsoft.ML.Samples.Dynamic.Trainers.BinaryClassification.Calibrators
```

Comment:
>Microsoft.ML.Samples.Dynamic.Trainers.BinaryClassification.Calibrators [](start = 10, length = 70)

please Microsoft.ML.Samples.Dynamic 

no need to be too fancy

Reply:
I don't know. I told same thing to @shmoradims and he told me I'm silly. I don't want be part of any fights regarding samples and how we organize them. So I would just follow what he is doing.

---
In reply to: [261054779](https://github.com/dotnet/machinelearning/pull/2766#discussion_r261054779) [](ancestors = 261054779)